in.varb.Ausweiskopie.yaml

```diff
@@ -0,0 +1,114 @@
+app-id: "in.varb.Ausweiskopie"
+runtime: "org.freedesktop.Platform"
+runtime-version: "24.08"
+sdk: "org.freedesktop.Sdk"
+command: ausweiskopie
+finish-args:
+  - --socket=x11
+  - --share=ipc
+modules:
+  # Step 1.1: Requirements for Tcl/Tk with proper font rendering.
+  # Freetype renders font, and fontconfig finds them.
+  - name: "freetype"
+    sources:
+      - type: archive
+        url: https://download.savannah.gnu.org/releases/freetype/freetype-2.13.3.tar.xz
+        sha256: 0550350666d427c74daeb85d5ac7bb353acba5f76956395995311a9c6f063289
+        x-checker-data:
+          type: html
+          url: "https://download.savannah.gnu.org/releases/freetype/"
+          version-pattern: "freetype-([\\d\\.]+\\d).tar.xz"
+          url-template: "https://download.savannah.gnu.org/releases/freetype/freetype-$version.tar.xz"
+
+  - name: "fontconfig"
+    sources:
+      - type: archive
+        url: https://www.freedesktop.org/software/fontconfig/release/fontconfig-2.16.0.tar.xz
+        sha256: 6a33dc555cc9ba8b10caf7695878ef134eeb36d0af366041f639b1da9b6ed220
+        x-checker-data:
+          type: html
+          url: "https://www.freedesktop.org/software/fontconfig/release/"
+          version-pattern: "fontconfig-([\\d\\.]+\\d).tar.xz"
+          url-template: "https://www.freedesktop.org/software/fontconfig/release/fontconfig-$version.tar.xz"
+  # Step 1.2: Install a current version of Tcl/Tk
+  - name: "tcl"
+    sources:
+      - type: archive
+        url: http://prdownloads.sourceforge.net/tcl/tcl8.6.16-src.tar.gz
+        sha256: 91cb8fa61771c63c262efb553059b7c7ad6757afa5857af6265e4b0bdc2a14a5
+        x-checker-data:
+          type: html
+          url: "https://sourceforge.net/projects/tcl/rss"
+          version-pattern: "https://sourceforge.net/.+/tcl(8\\.6\\.[\\d\\.]*\\d)-src.tar.gz/download"
+          url-template: "http://prdownloads.sourceforge.net/tcl/tcl$version-src.tar.gz"
+    subdir: unix
+    post-install:
+      - "chmod +w ${FLATPAK_DEST}/lib/libtcl8.6.so"
+  - name: "tk"
+    sources:
+      - type: archive
+        url: http://prdownloads.sourceforge.net/tcl/tk8.6.16-src.tar.gz
+        sha256: be9f94d3575d4b3099d84bc3c10de8994df2d7aa405208173c709cc404a7e5fe
+        x-checker-data:
+          type: html
+          url: "https://sourceforge.net/projects/tcl/rss"
+          version-pattern: "https://sourceforge.net/.+/tk(8\\.6\\.[\\d\\.]*\\d)-src.tar.gz/download"
+          url-template: "http://prdownloads.sourceforge.net/tcl/tk$version-src.tar.gz"
+    subdir: unix
+    post-install:
+      - "chmod +w ${FLATPAK_DEST}/lib/libtk8.6.so"
+
+  # Step 2: Get a new python installation with our Tk version
+  - name: "python"
```

Comment:
python 3.12 is in the runtime.

Reply:
why do you need something more recent?

Reply:
Nvm. "just" providing a recompiled Tk/Tcl is enough, and Python will pick it up automatically.